src/Services/WafDetectionService.php

```
@@ -0,0 +1,104 @@
+    public static function probe(bool $force_refresh = false): array
+    {
+        if (!$force_refresh) {
+            $cached = get_transient(self::TRANSIENT_KEY);
+            if (is_array($cached)) {
+                return $cached;
+            }
+        }
+
+        $result = ['blocked' => false, 'waf' => 'unknown'];
+
+        $probe_url = rest_url('slimstat/v1/settings-probe');
```

PHPStan (2.1.46)

[error] Function rest_url not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)